### DISCUSSION TITLE
Enhancement:  Change Field Group Assignment

### DISCUSSION THREAD

**Discussion by richhayler** [craftcms/cms#8693] [Ideas]
Currently if have a field and want to change its field group assignment you need to access the individual field settings and select the desired group from a dropdown. When you do this and click save you are then presented with the listing of all the fields in the selected group.

I’m proposing this behaviour should change as in my experience what the user actually wants is to be sent back to the original field group listing so they can repeat the above process for another field.

Taking this a bit further, I would expect the above pattern could be improved by providing some better options to batch apply operations to a number of fields simultaneously. For example, an option to select multiple fields and reassign then to a different field group.

**Comment by KatieMFritz** [comment]
Batch moving fields into different groups would be awesome - I've been spending a lot of time doing that one by one lately.

Or maybe @engram-design could build that into Field Manager?
